db/migrate/20240721151707_make_category_id_not_null_in_products.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class MakeCategoryIdNotNullInProducts < ActiveRecord::Migration[7.0]
```

Comment:
Duplicate with this change in other PR [LOC](https://github.com/imwillim/clone_tch/blob/59e87fea70443e710aac78e9e3d874df3327aed3/db/migrate/20240721115247_add_category_id_to_products.rb#L5)

Reply:
Removed it, [TCH-7](https://github.com/imwillim/clone_tch/pull/9/commits/bc4c547648afa9a4a59eeb18939a3f3e3b833ab1#diff-1c44946f848b5648d8ab62d2827b3fd8ff43cf3441e7f012d10b0683ebfbb702) has reference not null